web-frontend/src/assets/util.js

```
@@ -0,0 +1,587 @@
+
+	/**
+	 * Moves elements to/from the first positions of their respective parents.
+	 * @param {jQuery} $elements Elements (or selector) to move.
+	 * @param {bool} condition If true, moves elements to the top. Otherwise, moves elements back to their original locations.
+	 */
+	$.prioritize = function($elements, condition) {
+
+		var key = '__prioritize';
+
+		// Expand $elements if it's not already a jQuery object.
+			if (typeof $elements != 'jQuery')
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: "jquery" is not a valid type name

not a valid type name

(lint/suspicious/useValidTypeof)